# Topic to listen to for buzzer activations.
buzzer_topic: '/rpi_piezo_buzzer/activate'

# GPIO pin that the buzzer is connected to.
buzzer_gpio_pin: 27

# Duration of time (s) that the buzzer should operate.
buzzer_duration: 0.1

# Duration of time during operation that the buzzer should be 'on'.
buzzer_on_duration: 0.005

# Duration of time during operation that the buzzer should be 'off'.
buzzer_off_duration: 0.005
